engine/src/physics/solver/penetration_constraint/plugin.cpp

```diff
@@ -50,15 +50,12 @@ void solvePenetrationConstraint(Query<Entity, const Mass&, AccumulatedCorrection
                 penetration += glm::dot(deltaPenetration, constraint.normal);
             }
 
-            // penetration is positive but we need negative value for separation
-            penetration = -penetration;
-
             float bias = 0.0F;
             float massScale = 1.0F;
             float impulseScale = 0.0F;
             if (penetration > 0.0F)
             {
-                bias = penetration * 1.0F / subDeltaTime;
+                bias = 0.2F * penetration * (1.0F / subDeltaTime);
```

Comment:
Why `0.2F`? Should this be a constant somewhere? And should it be configurable by the dev?

Reply:
It would probably be a good idea to make all magic numbers configurable settings, not really for the user, but for our fine tuning work later on.

Reply:
Going to open an issue for this.